pkg/grpc/ksm.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package ksm;
```

Comment:
nit: early days - but I presume we will add a high level description in here.
I only notice as I think there is a place in `ksm.pb.go` where it would be copied to on generation.